roslaunch_editor/www/index.html

```diff
@@ -0,0 +1,140 @@
+<html>
+	<head>
+		<script src="roslib.js"></script>
```

Comment:
`roslib.js` will probably be used in all web-based plugins, how about moving it to `roswww_static`?

Reply:
What do you mean, `/roswww_static/roslib.js`? What about different versions might be needed? Also, I'm thinking on not requiring `roswww_static` for this. Plus, for now, it's compatible with `roswww`, too.